applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -0,0 +1,499 @@
+
+* :file:`child_secure_partition_manager.conf`
+  This configuration file contains the project-specific configuration for the :ref:`secure_partition_manager` child image.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/nrf9160/serial_lte_modem`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Testing
+=======
+
+The testing instructions focus on testing the application with a PC client.
+If you have an nRF52 Series DK running a client application, you can also use this DK for testing the different scenarios.
+
+|test_sample|
+
+1. |connect_kit|
+#. :ref:`Connect to the kit with LTE Link Monitor <lte_connect>`.
+   If you want to use a different terminal emulator, see `Connecting with a PC`_.
+#. Reset the kit.
+#. Observe that the nRF9160 DK sends a ``Ready\r\n`` message on UART.
+#. Send AT commands and observe the responses from the nRF9160 DK.
+   See the following sections for typical test cases.
+
+
+Testing generic AT commands
+---------------------------
+
+Complete the following steps to test the functionality provided by the :ref:`SLM_AT_gen`:
+
+1. Retrieve the version of the serial LTE modem application.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLMVER**
+      #XSLMVER: 1.2
+      OK
+
+#. Read the current baud rate.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLMUART?**
+      #SLMUART: 115200
+      OK
+
+   You can change the used baud rate with the corresponding set command, but note that LTE Link Monitor requires 115200 bps for communication.
+
+#. Retrieve a list of all supported proprietary AT commands.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XCLAC**
+      AT#XSLMVER
+      AT#XSLEEP
+      AT#XCLAC
+      AT#XSOCKET
+      AT#XSOCKETOPT
+      AT#XBIND
+      *[...]*
+      OK
+
+#. Check the supported values for the sleep command, then put the board in sleep mode.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLEEP=?**
+      #XSLEEP: (0, 1)
+      OK
+
+      **AT#XSLEEP=1**
+
+   Reset the board to exit sleep mode.
+   If you are testing with an external MCU and :option:`SLM_GPIO_WAKEUP` is enabled, you can wake up by GPIO as well.
+
+
+
+Testing TCP/IP AT commands
+--------------------------
+
+The following sections show how to test the functionality provided by the :ref:`SLM_AT_TCPIP`.
+
+TCP client
+~~~~~~~~~~
+
+1. Establish and test a TCP connection:
+
+   a. Check the available values for the XSOCKET command.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSOCKET=?**
+        #XSOCKET: (0, 1), (1, 2), (0, 1), <sec-tag>
+        OK
+
+   #. Open a TCP socket, read information (handle, protocol, and role) about the open socket, and set the receive time-out of the open socket to 30 seconds.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSOCKET=1,1,0**
+        #XSOCKET: 2, 1, 0, 6
```

Comment:
I get output #XSOCKET: 1, 1, 0, 6 when I test it?

Reply:
Same as me. It should be **#XSOCKET: 1, 1, 0, 6**